pkg/security/security_profile/profile/manager.go

```diff
@@ -625,7 +634,7 @@ func (m *SecurityProfileManager) LookupEventInProfiles(event *model.Event) {
 	// lookup profile
 	profile := m.GetProfile(selector)
 	if profile == nil || profile.Status == 0 {
-		m.eventFiltering[event.GetEventType()][NoProfile][NA].Inc()
+		m.incEventFilteringStat(event.GetEventType(), NoProfile, NA)
```

Comment:
nit, but I'd like to see Increment as a full word because we no longer have the func description upon hover